### ISSUE TITLE
Discord bot to check Team availability

### ISSUE THREAD

**Issue by markoprljic** [GeneralMagicio/gm-planning#64]

**Comment by VMLVaske** [comment]
It is not a discord bot command, but I have set up this: https://www.notion.so/giveth/24a713d38f754f07adb5adfcf1d931ac?v=8ac7a8f65d29406ab917b268cf9eb86f 

We need to maintain it manually. A discord bot would be nicer, but this visualizes things in a first instance. Still need to work out a way how to automate - but would need to figure that out for a discord bot as well.

**Comment by markoprljic** [comment]
Yes, I'm aware of this. I also know we are (or were) using other trackers (Freshelle and Nico) and have the #who-is-available channel on Discord. I also know that @tomislavmamic is aware of this and that we might be building a solution with Coda + Discord automation and bot commands with @Vyvy-vi but I cannot confirm this at the moment. 

This issue should address the above mentioned. For time being, the Notion overview is ok but we should figure out a plan and solution for something better. Ideally, we could tie this in to the onboarding and connect discord handles with the monthly availability from the agreement + time off reporting + any engagement on existing projects that reduces their availability at the time of requesting the information. 

I know I said a lot :D

**Comment by Vyvy-vi** [comment]
SInce #who-is-available is being used as the default place to add availability info, we can set up the following commands-
`/unavailable time-slot, REASON`
`/available time-slot, REASON`

the data from here can go somewhere in Coda, from where we can _maybe_ send DMs to any buddies/project leads about the same (I think we already have a "project team"-like section in Coda)

**Comment by markoprljic** [comment]
Yeah, we have Time-off in Coda already, we just need to figure out where the info is added first, is it in who-is-available channel or Coda? Since Coda is considered single source of truth maybe best to pull the info from there.  
I think we should just check the current process on how/where team members add their availability.
